main_test.go

```diff
@@ -124,29 +127,31 @@ func runInternalJsonnet(i jsonnetInput) jsonnetResult {
 	vm.NativeFunction(jsonToString)
 	vm.NativeFunction(nativeError)
 
-	var output string
-
 	rawOutput, err := vm.evaluateSnippet(i.name, string(i.input), i.eKind)
-	var isError bool
-	if err != nil {
+	switch {
+	case err != nil:
 		// TODO(sbarzowski) perhaps somehow mark that we are processing
 		// an error. But for now we can treat them the same.
-		output = errFormatter.Format(err)
-		output += "\n"
-		isError = true
-	} else {
-		output = rawOutput.(string)
-		isError = false
-	}
-
-	return jsonnetResult{
-		output:  output,
-		isError: isError,
+		return jsonnetResult{
+			output:  errFormatter.Format(err) + "\n",
+			isError: true,
+		}
+	case i.eKind == evalKindMulti:
+		return jsonnetResult{
+			outputMulti: rawOutput.(map[string]string),
+		}
+	default:
+		return jsonnetResult{
+			output: rawOutput.(string),
+		}
 	}
 }
 
 func runJsonnetCommand(i jsonnetInput) jsonnetResult {
 	// TODO(sbarzowski) Special handling of errors (which may differ between versions)
+	if i.eKind != evalKindRegular {
+		panic(fmt.Sprintf("eKind must be evalKindRegular for jsonnet CLI testing; was %v", i.eKind))
```

Comment:
Ah, hmm. It looks like runJsonnetCommand is actually completely broken at the moment: the goldens are completely different, since they're missing the line of dashes and the filename is <stdin> from the CLI.

I'm going to defer this for now and add a TODO,